lib/elixir/test/elixir/test_helper.exs

```diff
@@ -1,4 +1,10 @@
-ExUnit.start [trace: "--trace" in System.argv]
+exclude =
+  case :erlang.system_info(:otp_release) do
```

Comment:
> Yes, one step at a time. And this are tests, we can afford to be flexible and change later on.

OK lets stick with `:otp_release` I was just trying to drive home the point that this kind of version checking is often inadequate.